server/lomas_server/dp_queries/dummy_dataset.py

```diff
@@ -37,9 +36,9 @@ def make_dummy_dataset(
     Returns:
         pd.DataFrame: dummy dataframe based on metadata
     """
-    # Setting seed
-    random.seed(seed)
-    np.random.seed(seed)
+    # pylint: disable=too-many-locals
+    # Creating new random generator with fixed seed
+    rng = np.random.default_rng(seed)
```

Comment:
It seems better like this but I don't know why ?

Reply:
Numpy deprecated the use of np.random.xyz in favour of using a RandomNumberGenerator, hence why I changed it. Also I removed the dependency on the python random package so that we only use one source of randomness.

Reply:
nice, thank you